### PR TITLE
Removed deprecated backticks with repr

### LINKED ISSUE #846: Remove deprecated backticks from Python code
https://github.com/internetarchive/openlibrary/blob/f8092840a77c7479a352fd83fd068d340f97d3e3/openlibrary/solr/inside/index_all.py#L29

The single reversed comma quotation marks (&#8219;) or whatever that is here, and repeated lower in this module, seem anomalies. Is it a typo, or a real thing?

### PR DESCRIPTION
**Pull request by salman-bhai**
Closes #846 

## Description:
In this Pull Request we have made the following changes:
1. Backticks removed and replaced with repr

### CODE REVIEW

File: openlibrary/catalog/update_count.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 def commify(n):
     """
-Add commas to an integer `n`.
+Add commas to an integer repr(n).
```
Comment (hornc) [inline openlibrary/catalog/update_count.py]: @salman-bhai , @mekarpeles I see this has been merged, but repr() isn't appropriate here in a docstring, should just be normal single quotes

File: scripts/z3950_view.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def GET(self, ia):
         if from_marc_xml:
             ret += '<h2>From MARC binary on archive.org</h2>'
 
-            ret += 'record length: ' + `len(meta_mrc)` + ' bytes<br>'
+            ret += 'record length: ' + repr(len(meta_mrc)) + ' bytes<br>'
```
Comment (hornc) [inline scripts/z3950_view.py]: It seems many of these `repr()` are unnecessary. it's not clear what they are adding `repr(<int>)`?

File: openlibrary/catalog/amazon/import.py [hunk 3]
```diff
@@ -166,7 +166,7 @@ def import_file(filename):
             # TODO load book
             continue
         #print asin, found
-        #print `edition['title'], edition.get('subtitle', None), edition.get('flags', None), edition.get('binding', None)`
+        #print(repr(edition['title'], edition.get('subtitle', None), edition.get('flags', None), edition.get('binding', None)))
```
Comment (hornc) [inline openlibrary/catalog/amazon/import.py]: Many of these commented out print statements should probably be removed.
Unless the code is in the scripts folder I'd suggest we remove all debugging `prints` commented out or not. If the message is important it should be passed to logging. Maybe that's for a different issue (or while people are working in an area)

File: openlibrary/catalog/update_count.py (hunk 1 above)
Comment (tfmorris) [reply openlibrary/catalog/update_count.py]: My preferences, in order, are n, N, 'n', "n". I don't think the quotes are needed for a single character variable name.

File: scripts/z3950_view.py (hunk 2 above)
Comment (tfmorris) [reply scripts/z3950_view.py]: Yes, just len(meta_mrc). If there's some weird edge case where this doesn't work str(len())

File: openlibrary/catalog/amazon/import.py (hunk 3 above)
Comment (tfmorris) [reply openlibrary/catalog/amazon/import.py]: Definitely a separate issue. This should be: a) part of the coding guidelines, and b) the developer guidelines should say to bring code being worked on up to standard with the coding guidelines (in a separate commit!).

I don't think it's a good idea to go through and do a wholesale cleanup.

Comment (salman-bhai) [reply openlibrary/catalog/amazon/import.py]: Yes so I wasn't sure if what to do with such print statements. Which are commented out! Maybe we could look at module wise review of removing debugging prints?

File: scripts/z3950_view.py (hunk 2 above)
Comment (salman-bhai) [reply scripts/z3950_view.py]: I wasn't sure of this as well! What should be done here?

File: openlibrary/catalog/update_count.py (hunk 1 above)
Comment (salman-bhai) [reply openlibrary/catalog/update_count.py]: I'll change this in a separate PR! Will go through this again and get it done!

File: scripts/z3950_view.py (hunk 2 above)
Comment (salman-bhai) [reply scripts/z3950_view.py]: @hornc can you probably tell where all are the `repr` unnecessary. I'll replace all of them in a different PR.

**Pull request merged**